server/routes/nl/helpers.py

```diff
@@ -66,10 +66,16 @@ def parse_query_and_detect(request: Dict, app: str, debug_logs: Dict):
   nl_bad_words = current_app.config['NL_BAD_WORDS']
 
   test = request.args.get(params.Params.TEST.value, '')
+  # i18n param
   i18n_str = request.args.get(params.Params.I18N.value, '')
   i18n = i18n_str and i18n_str.lower() == 'true'
+  # use default place param
   udp_str = request.args.get(params.Params.USE_DEFAULT_PLACE.value, 'true')
   udp = udp_str and udp_str.lower() == 'true'
```

Comment:
Do we want to deprecate USE_DEFAULT_PLACE ?  Add TODO?

Reply:
Done